### ISSUE TITLE
Allow for value tittles in room name.

### ISSUE THREAD

**Issue by tomtom13** [marcokreeft87/room-card#77]
As per tittle - it would be nice to be able to include names for values in the room name. At this moment if one has 3 temperature sensors in the room, only 3 seperate values will show, while having a tittle above them like C 1 could help.
While we're at it, it would be also nice to remove units from the same values to get stuff more densely packed.

**Comment by marcokreeft87** [comment]
I don't get it. Can you give an example?

**Comment by tomtom13** [comment]
Units are included with value, and there is no wan to configure name that would sit above them:
<img width="404" alt="Screenshot 2022-09-29 at 22 17 16" src="https://user-images.githubusercontent.com/22304458/193143827-e0187a4e-9397-48f1-921c-6a85137dcec5.png">

While it's possible with other plugins (arbitrary NOT in the name of the room) 
<img width="396" alt="Screenshot 2022-09-29 at 22 17 22" src="https://user-images.githubusercontent.com/22304458/193144047-9b5ef66a-342a-4404-9cb4-8b01f2602b07.png">

**Comment by marcokreeft87** [comment]
> Units are included with value, and there is no wan to configure name that would sit above them:
> <img width="404" alt="Screenshot 2022-09-29 at 22 17 16" src="https://user-images.githubusercontent.com/22304458/193143827-e0187a4e-9397-48f1-921c-6a85137dcec5.png">
> 
> While it's possible with other plugins (arbitrary NOT in the name of the room) 
> <img width="396" alt="Screenshot 2022-09-29 at 22 17 22" src="https://user-images.githubusercontent.com/22304458/193144047-9b5ef66a-342a-4404-9cb4-8b01f2602b07.png">
> 

If this is what you need, you might wanna use this card https://github.com/benct/lovelace-multiple-entity-row

**Comment by tomtom13** [comment]
OK, but the room card allows to have values next to room name. I was hoping for more configuration,

**Comment by marcokreeft87** [comment]
But how can a room have multiple temperatures?

**Comment by tomtom13** [comment]
Actually I have 3 different cases where room can have up to 4 C variation easily:
- room is big and heating / ac is more localised than global, so monitoring those temps allows to make something about it realtime and think of long term solution. 
- room has bad heat insulation and external weather influence is different in different parts of the room
- room has an equipment concentrated in one location, and heat generated provides significant difference.

There are also some corner cases like ensuite. But this can be solved as a separate room.

**Comment by marcokreeft87** [comment]
Ah like that. I'd advice to put the temperature entities in de entities and not in the info_entities in this case. That way you can put the name above it and let it show state.

**Comment by tomtom13** [comment]
Yes I'm already doing that, but because of that my cards are large, and was hoping I can compact stuff with your card - where I dump more data next to the room name. I honestly think that HS is suffering from bad dashboard layout, and swaths of display are wasted making is a scroll fest. I do understand that packing things too tightly is not a good choice, but leaving large unused spaces that users can't facilitate is also not good.
